sorc/build_all.sh

```diff
@@ -37,6 +37,8 @@ Usage: ${BASH_SOURCE[0]} [-a UFS_app][-c build_config][-d][-f][-h][-j n][-v][-w]
     Execute all build scripts with -v option to turn on verbose where supported
   -w:
     Use structured wave grid
+  -y:
+    Use hystrostatic version of FV3
```

Comment:
```suggestion
    Use hydrostatic version of FV3
```

Reply:
@NeilBarton-NOAA This typo hasn't been fixed.